src/platform/client.py

```diff
@@ -117,18 +117,7 @@ def __init__(
                 Defaults to False.
         """
 
-        # Environment variables ALWAYS override explicit parameters and skip disk reads
-        if ENV_VARIABLES["access_token"] in os.environ:
-            token = os.environ[ENV_VARIABLES["access_token"]]
-        elif token is None:
-            token = get_token()
-
-        if ENV_VARIABLES["org_key"] in os.environ:
-            org_key = os.environ[ENV_VARIABLES["org_key"]]
-        elif org_key is None:
-            org_key = get_value()["org_key"]
-
```

Comment:
we're short circuiting this -- if env=local, we're configuring the client automatically